### DISCUSSION TITLE
How to upload to tiddlyhost from a nodejs wiki?

### DISCUSSION THREAD

**Discussion by linonetwo** [simonbaird/tiddlyhost#319] [Q&A]
A member in QQ group ask how to update tiddlyhost when using TidGi (nodejs wiki) with a button @simonbaird .

And I also want to ask if it is possible to exclude some tiddlers (for example, `[tags[Private]] [tags[Calendar]]`) when do such upload.

**Comment by linonetwo** [comment]
I found I can programmatically [download](https://github.com/simonbaird/tiddlyhost/blob/main/examples/thost-downloader) / [upload](https://github.com/simonbaird/tiddlyhost/wiki/Saving-to-a-Tiddlyhost-site-with-a-command-line-script) from HTML format. So I can export HTML as I wish, to a /temp folder, then do this.

**Comment by simonbaird** [comment]
To be honest I don't have much experience using TiddlyWiki with the node server, so don't know the answer. But the process would be simple enough if you can produce a single file version of the TiddlyWiki, (which it seems like you have discovered a solution for).
